### DISCUSSION TITLE
Not available episode-transcripts

### DISCUSSION THREAD

**Discussion by eshimischi** [pointfreeco/pointfreeco#715] [Q&A]
https://github.com/pointfreeco/pointfreeco/blob/8ec57ed8e83ca8ca35a9388af61068430b0c129b/.gitmodules#L3

this repo is unavailable

**Comment by mbrandonw** [comment]
That is a private repo of our transcripts. It isn't needed to run the project locally though. If you follow the steps from [getting started](https://github.com/pointfreeco/pointfreeco#getting-started) you shouldn't need to clone that repo.
